docs/REST API/rest-api.md

```diff
@@ -9706,6 +9706,7 @@ Update a package to install on macOS, Windows, or Linux (Ubuntu) hosts.
 | id | integer | path | ID of the software title being updated. |
 | software        | file    | form | Installer package file. Supported packages are .pkg, .msi, .exe, .deb, and .rpm.   |
 | team_id         | integer | form | **Required**. The team ID. Updates a software package in the specified team. |
+| categories        | array | form | Category names, as shown in the UI. Sets the software package to the specified categories. |
```

Comment:
```suggestion
```

Dropping this out as #27681 covers API changes, and we don't want REST API changes to hit `main` pre-release.